electrumsv/network.py

```diff
@@ -155,6 +155,9 @@ async def _synchronise_headers_for_server_tip(self, server_state: HeaderServerSt
                 server_tip=tip_header)
             heights = [height for height in range(any_common_base_header.height,
                 tip_header.height + 1)]
+            if len(heights) > 2000:
+                logger.warning("Synchronizing %s headers. Wallet functionality"
+                    "will be temporarily limited until complete", len(heights))
```

Comment:
Perhaps it should instead read:

`The Node JSON-RPC API functionality will be temporarily limited until complete`

Reply:
The intention of adding this logging was to indicate to users who look at the (non-debug) output, why the wallet might be slow. So write something like "Synchronizing %s headers. This may cause the wallet to be unresponsive or slow." ?